bebraven/platform
# Add some padding the Projects and Surveys when shown in their own window

### Description

Task: https://app.asana.com/0/1174274412967132/1198943782093721

This is the behavior on the Canvas mobile app. You launch the LTI assignment
into a window of it's own. The text was smooshing all the way up against the
edge of the screen with no padding.

This change just adds a bit of padding.

#### Test Plan:
- View a Project in a window of it's own on a small screen size.
- View a Project inline in Canvas on a small screen size. It should still look good.
- View a Project submission as a TA on a small screen size (still in Canvas).
- Do the above for Impact Surveys too.

### Screenshots (if appropriate):
Project
<img width="300ox" src="https://user-images.githubusercontent.com/5596986/112854945-5917ab80-907c-11eb-9417-7122d18318bc.png">

Impact Survey
<img width="300px" src="https://user-images.githubusercontent.com/5596986/112855086-7d738800-907c-11eb-9813-43aa73362de7.png">



### Types of changes




### Checklist:
